site/content/en/docs/contributing/running-tests.md

```diff
@@ -223,3 +223,21 @@ curl -L -o opa https://openpolicyagent.org/downloads/v0.45.0/opa_linux_amd64_sta
 chmod +x ./opa
 ./opa test cvat/apps/iam/rules
 ```
+
+### Linting Rego
+
+The Rego policies in this project are linted using [Regal](https://github.com/styrainc/regal).
+
+- In a Docker container
+```bash
+docker run --rm -v ${PWD}/cvat/apps/iam/rules:/rules \
+    ghcr.io/styrainc/regal:0.11.0 \
+    lint /rules
```

Comment:
```suggestion
    lint /rules --ignore-files *_test.gen.rego
```

Reply:
Ah yes, I remember trying that as part of this integration, but I didn't realize they'd be generated at this point. There's an `ignore` directive available for the config file though — it would probably be cleaner using that, or WDYT? I'll submit an update with that change.

Reply:
I agree, it will be better to put it into the configuration file.

Reply:
Just pushed a change adding this to `.regal/config.yaml`:
```yaml
ignore:
  files:
    - "*_test.gen.rego"
```
That makes it less cumbersome to run the command from the CLI. Or do you prefer that more explicit option?

Reply:
Oh, sorry, I think we wrote about at the same time :) All good then! I ran the command to verify, and it does not stall on the test files:
```
regal lint --format=github cvat/apps/iam/rules
22 files linted. No violations found.
```